tests/unit/src/test/scala/tests/JavaDefinitionSuite.scala

```diff
@@ -107,18 +110,32 @@ class JavaDefinitionSuite extends BaseLspSuite("java-definition") {
       val deps = dependencies
         .map(s => s""""$s"""")
         .mkString("[", ", ", "]")
-      val layout =
+      def simpleLayout =
         s"""|/metals.json
             |{
             |  "a": {
             |    "libraryDependencies": $deps
             |  }
             |}
             |""".stripMargin
+
+      def workspaceFoldersLayout =
```

Comment:
Why change the JavaDefinitionSuite, it doesn't seem related.

Reply:
when run with virtual files it is

Reply:
ach ok!